### PR TITLE
Update size.md

### PR DESCRIPTION
**Pull request by trolologuy**

### CODE REVIEW

**Comment by Crell** [comment]
We don't link to the community site from the docs currently, and I'd rather keep it that way.  The docs should be a mostly self-contained resource.  If there's a need to inline that tutorial in the docs, the let's just inline it.

**Comment by trolologuy** [comment]
I inlined the minimal version of the community site's content.

File: docs/src/configuration/app/size.md [hunk 1]
```diff
@@ -20,6 +20,29 @@ If the total resources requested by all apps and services is larger than what th
...
+Click those dots to pull up a drop-down menu and select edit to modify that project’s configuration.
+
+2. Update the plan's settings according to you needs
+
+From there you'll be able to update several settings such as the:
+- plan size
+- number of environments
+- Environments application size
+- storage sizing
+- number of Users
```
Comment (Crell) [inline docs/src/configuration/app/size.md:36]:
```suggestion
From there you'll be able to update several settings such as the:

* plan size
* number of environments
* development environments application size
* storage sizing
```

File: docs/src/configuration/app/size.md [hunk 2]
```diff
@@ -20,6 +20,29 @@ If the total resources requested by all apps and services is larger than what th
 
 Note that in a development environment this value is ignored.  Service containers are always set to size `S` in development environments.  The application container is always set to the "Development Application Size" as configured on the plan.  That defaults to `S` but you can increase it on your plan settings page for a fee.
 
+## How to modify my Platform.sh project plan and its resources
+
+1. [Login to your account](https://accounts.platform.sh/user) to view the accessible projects.
+For the projects that you are the Account Owner for, you will see three vertical dots. 
+Click those dots to pull up a drop-down menu and select edit to modify that project’s configuration.
```
Comment (Crell) [inline docs/src/configuration/app/size.md:27]:
```suggestion
1. [Login to your account](https://accounts.platform.sh/user) to view the accessible projects.  For the projects that you are the Account Owner for, you will see three vertical dots.  Click those dots to pull up a drop-down menu and select edit to modify that project’s configuration.
```

File: docs/src/configuration/app/size.md [hunk 3]
```diff
@@ -20,6 +20,29 @@ If the total resources requested by all apps and services is larger than what th
...
+From there you'll be able to update several settings such as the:
+- plan size
+- number of environments
+- Environments application size
+- storage sizing
+- number of Users
+
+The pricing will be displayed and updated according to the changes.
+
+If you did some changes that you wish to apply, click the Update Plan button on the bottom of the page.
```
Comment (Crell) [inline docs/src/configuration/app/size.md:40]:
```suggestion
If you made a change that you wish to apply, click the Update Plan button on the bottom of the page.
```

**Pull request merged**